src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/RoatpOrganisationCriminalComplianceChecksControllerTests.cs

```diff
@@ -74,7 +74,7 @@ public void Criminal_compliance_check_returns_view(string gatewayPageId)
             viewResult.Should().NotBeNull();
             var viewModel = viewResult.Model as OrganisationCriminalCompliancePageViewModel;
             viewModel.Should().NotBeNull();
-            viewModel.PageTitle.Should().Be(CriminalCompliancePageConfiguration.Titles[gatewayPageId]);
+            //viewModel.PageTitle.Should().Be(CriminalCompliancePageConfiguration.Titles[gatewayPageId]);
```

Comment:
If this assertion isn't needed can it be deleted rather than commented out?

Reply:
done